organisation.rst

```diff
@@ -1,53 +1,111 @@
-Hackspace Organisation
+Hackspace Organisation and Teams
 ======================
 
-Nottingham Hackspace is registered as a non-profit limited company (Reg. No. 07766826). If there is any surplus after rent and other expenses have been paid, it is spent on tools and consumables.
+Nottingham Hackspace is registered as a non-profit limited company (Reg. No. 07766826). Surplus after rent and other expenses are used for tools, consumables, and Hackspace improvements.
 
-There is a seven person board of trustees elected to oversee the hackspace, ensure financial sustainability and vote on critical issues. All of the board are volunteers. **Members' meetings are held publicly, in the hackspace at 20:00 on the first Wednesday of each month.** These meetings are open to all Nottinghack members. Votes conducted at meetings are also open to all members.
+An elected Board of Trustees comprised of seven people oversee Nottingham Hackspace; ensuring financial sustainability and managing critical issues. All of the board are volunteers, Nottinghack does not have any paid employees or membership tiers. **Members' meetings are held in the Hackspace at 20:00, on the first Wednesday of each month.** These meetings, and votes conducted at the meetings, are open to all members. 
 
-Please find details of the board members here: http://wiki.nottinghack.org.uk/wiki/Board
+Please find details of the board members here: http://wiki.nottinghack.org.uk/wiki/Board. Board elections take place each May, any member can stand for election.
 
-Board elections take place each May, any member can stand for election. There's more information available about our constitution on the wiki:
-
-https://wiki.nottinghack.org.uk/wiki/Constitution
+Our constitution is also available on the wiki: https://wiki.nottinghack.org.uk/wiki/Constitution
 
 Teams
 -----
-The day-to-day running of the hackspace is done by you and every member.
-The various activities are run by teams, which are outlined here:
+The day-to-day running of the Hackspace is done by every one of our members. Even quick and simple things like tidying up or putting tools back into their proper storage are a great help.
+
+More specific activites are managed by teams within the Hackspace, a list of which can be found below. Please remember the Hackspace is a community and not a service; we can't guarantee tool uptime or material availability. The teams are all members like yourself, who volunteer their time to help make the space better.
+
+Want to get involved in a team? Drop them an email, or ask them when they're in the space.
 
-Safety
+Safety - safety@nottinghack.org.uk
```

Comment:
Need to extend the backticks to cover the entire title

I'm going to stop adding this comment - it needs changing on all titles on this page